dev/com.ibm.ws.jsp.2.3_fat/fat/src/com/ibm/ws/jsp23/fat/tests/JSP23JSP22ServerTest.java

```diff
@@ -112,4 +114,44 @@ public void testJspFeatureChange() throws Exception {
         assertTrue("The response did not contain: JSP version: 2.2", response.getText().contains("JSP version: 2.2"));
 
     }
+
+    /**
+     * Test a JSP request with the jsp-2.3 feature enabled.
+     * Then change the feature to pages-3.0 and request the same JSP again.
```

Comment:
Should this documentation be a bit different here? It seems you're driving a request with pages-30 first then jsp-2.3?